frontend/javascripts/oxalis/view/layouting/flex_layout_wrapper.js

```diff
@@ -68,6 +68,8 @@ type State = {
   model: Model,
 };
 
+const ignoredLayoutChangesByAnalytics = ["FlexLayout_SetActiveTabset"];
```

Comment:
I think, the following scenarios shouldn't send an event, either:

- toggling the side bars
- switching tabs (in the sidebar, the event is already ignored. but when moving XY and XZ next to each other, switching the tabs sends events)

Reply:
But moving the viewport around and resizing them should still send the event? Or should the event only be send when switching between layouts?

Reply:
> I think it depends on what exactly we want to track / use this tracking event for.

Yes, definitely. I don't have a strong opinion about what we really want to track, but I'm leaning towards tracking only stuff which is not "obvious" (e.g., it seems obvious that users will click on tabs to switch between them). With regard to maximizing, resizing and switching layouts, I'm not too sure that this is obvious for all users. Therefore, these events seem more interesting to me. However, in the end, this is also not very high-pri / not worth delving into how to filter the events best. So, maybe opt for the easier route if it makes sense (e.g., only track switches between layouts).